connector/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaBatchPartitionReader.scala

```diff
@@ -34,9 +36,17 @@ private[kafka010] case class KafkaBatchInputPartition(
     failOnDataLoss: Boolean,
     includeHeaders: Boolean) extends InputPartition
 
-private[kafka010] object KafkaBatchReaderFactory extends PartitionReaderFactory {
+private[kafka010] object KafkaBatchReaderFactory extends PartitionReaderFactory with Logging {
   override def createReader(partition: InputPartition): PartitionReader[InternalRow] = {
     val p = partition.asInstanceOf[KafkaBatchInputPartition]
+
+    val queryId = TaskContext.get().getLocalProperty(StreamExecution.QUERY_ID_KEY)
+    val batchId = TaskContext.get().getLocalProperty(MicroBatchExecution.BATCH_ID_KEY)
+    logInfo(s"Creating Kafka reader partitionId=${TaskContext.get().partitionId()} " +
```

Comment:
minor: placing partitionId near to topicPartition may lead to misunderstand that partitionId refer to Kafka's one (hence redundant info.). It's Spark's partition ID. 

I'd suggest placing it after taskId, and maybe add ` for the query ` in between `untilOffset` and `queryId` to differentiate Kafka related info vs Spark/query related info.